ts/api/api.ts

```diff
@@ -123,6 +124,11 @@ export interface OpenYoloApi {
    * Cancels the last pending OpenYOLO request.
    */
   cancelLastOperation(): Promise<void>;
+
+  /**
+   * Sets the desired log level for the OpenYOLO library.
+   */
+  setLogLevel(level: LogLevel): Promise<void>;
```

Comment:
Oh I see what you mean. It's true if you use the SPI code in TS, but after compilation into JS, only the objects exported on window will be accessible. So for now, it's only the `ProviderFrame.initialize` function and the new `Error` object.